models/articles.models.js

```diff
@@ -1,5 +1,28 @@
 const db = require("../db/connection.js");
 
+exports.fetchArticles = () => {
+  return db
+    .query(
+      `SELECT
+        a.author,
+        a.title,
+        a.article_id,
+        a.topic,
+        a.created_at,
+        a.votes,
+        a.article_img_url,
+        COUNT(comments.comment_id) as comment_count
```

Comment:
COUNT(comments.comment_id):: INT as comment_count
ensure that this comes back as a number